src/components/NavBar.jsx

```diff
@@ -13,12 +37,20 @@ function NavBar() {
     },
   ];
   return (
-    <nav>
-      <h1>Aman Bookstore</h1>
-      <ul>
+    <nav style={navStyle}>
+      <h1 style={h1Style}>
+        Amanuel Bookstore
+      </h1>
+      <ul style={ulStyle}>
         {links.map((link) => (
-          <li key={link.name}>
-            <NavLink to={link.path}>{link.name}</NavLink>
+          <li
+            key={link.name}
+            style={liStyle}
+          >
+            <NavLink to={link.path} style={navLinkStyle}>
+              {link.name}
+
+            </NavLink>
           </li>
         ))}
       </ul>
```

Comment:
- [ ] It would be nice if you could add the user icon on the right of the navbar so your design looks exactly like the original design. 👍🏽

Reply:
yeah but it's already placed at the right of the navbar